### ISSUE TITLE
A11y_VSCode_ChangingtheUsersettings_ScreenReader:Screen reader does not announce the expand, collapse state of 'auto' combo box.

### ISSUE THREAD

**Issue by bhagwatshelke1047** [microsoft/vscode#76691]
**Environment Details:**
VSCode Version : 1.36.1 (user setup)
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.3.1
Screen Reader Versions : NVDA2019.1.1

**Repro Steps:**
1.Open Visual studio code editor.
2.Open the command Pallet by pressing "F1" key and type "setting".
3.Using down-arrow key select user setting.
4.Type the word "Auto " and select one of the suggestion given by the vs code editor.
5.Type the "Tab size" and select the option from the auto suggestion.
6.Start screen reader

**Actual:**
 Screen reader does not announces the expand, collapse state of 'auto' combo box.

**Expected:**
Screen reader should announce the expand and collapse state of combo box.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
If the expand and collapse state of combo box are not announced then the  screen reader users will not get the exact information.

**MAS Reference:**
**MAS 1.3.1 -** https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
![A11y_VSCode_ChangingtheUsersettings_ScreenReader](https://user-images.githubusercontent.com/52443697/60713674-20927e80-9f37-11e9-8d0f-2f503ec5ec47.jpg)

**Comment by isidorn** [comment]
In your screenshot it can be seen that the "collpsed" is read out. It says "combo box auto collapsed".
Closing as designed.
Please reopen if you disagree and please provide arguments, thank you
